src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -160,11 +169,22 @@
     )
     private File outputSegmentsVCFFile;
 
+    @Argument(
+            doc = "Output denoised copy ratio file concatenated together from call shards.",
```

Comment:
Just "Output denoised copy ratio file." is probably fine.

Reply:
Done